### ISSUE TITLE
[QUESTION] Interest in taking over `fastapi-utils`?

### ISSUE THREAD

**Issue by dmontagu** [yuval9313/FastApi-RESTful#227]
I realize I have done a terrible job of maintaining that repository, and given the recent activity on this fork, I'd be happy to give the maintainers of this repository control over the `fastapi-utils` repo and/or transfer ownership (on PyPI etc.).

If you are interested let me know and I'll figure out a better way for us to get in contact than just commenting on a github issue.

If you want to just keep your fork separate that's fine too!

**Comment by yuval9313** [comment]
Yes, I will be happy to join `fastapi-utils` and help maintain that repo as well contact me so we could sort the detail out

**Comment by thomasaarholt** [comment]
Did you folks discuss this? Would be nice to know which package one should base off going forwards, though I get that you're both doing this in your free time :)

**Comment by yuval9313** [comment]
I'm sorry, I've been on a break for a while and missed @dmontagu invitation.

But I'm back now and ready to work 🤓

**Comment by wbarnha** [comment]
Bumping because it'd be nice to see both projects get updates. Let me know if you want help with CI/CD streamlining.

**Comment by vedpd11** [comment]
Waiting for the update on this. old fastapi-utils doesnt support new versions of fastapi and pydantic. Need to decide if I should switch or not.
fastapi-utils library had been life saver.

**Comment by dmontagu** [comment]
@yuval9313 apologies for letting this drag on so long, if you are still interested please email me at david@pydantic.dev and we can coordinate the handoff.
